src/test/unit/project-converter_test.ts

```diff
@@ -2485,6 +2485,42 @@ console.log('second script');
       });
     });
 
+
+    testName = 'copy and escape comments that include JS comment tags';
+    test(testName, async () => {
+      setSources({
+        'test.html': `
+<!-- /* First comment */ -->
+
+<script>
+  // comment in script
+  console.log('second script');
+</script>
+
+<!--
+  /**
+   *  Final comment
+   **/
+-->`
+      });
+
+      assertSources(await convert(), {
+        'test.js': `
+// comment in script
+/* /* First comment *\\/ */
```

Comment:
Seems odd that the first comment is now after the `// comment in script`

Reply:
Agreed. This is common across all our tests, I'll add a separate issue to tackle

Reply:
Agreed, filed as https://github.com/Polymer/polymer-modulizer/issues/281